android/filament-android/src/main/java/com/google/android/filament/Skybox.java

```diff
@@ -21,6 +21,31 @@
 
 import com.google.android.filament.proguard.UsedByReflection;
 
+/**
+ * Skybox
+ * <p></p>
+ * When added to a Scene, the Skybox fills all untouched pixels.
+ * <p></p>
+ * <h1><u>Creation and destruction</u></h1>
+ *
+ * A <code>Skybox</code> object is created using the {@link Skybox.Builder} and destroyed by calling
+ * {@link Engine#destroySkybox}.
+ * <p></p>
+ * <pre>
+ *  Engine engine = Engine.create();
+ *
+ *  IndirectLigh* skybox = new Skybox::Builder()
+ *              .environment(cubemap)
+ *              .build(engine);
+ *
```

Comment:
I think it's great that you added a usage example like this.  Maybe add `scene.setSkybox` here?